Updated to Debian11

Xitee1 opened the pull request.

Xitee1 committed:
Made changes to work with Debian11

Xitee1 committed:
Update to Debian11

cagnulein commented:
@Xitee1 i tried your patch but it didn't work

root@pvei7:~# bash -c "$(wget -qLO - https://raw.githubusercontent.com/Xitee1/proxmox_motioneye_container/master/create_container.sh)"


More than one storage locations detected.

1) backup
2) local
3) local-lvm
4) ssd256
5) ssd256_2
6) ssd480

Which storage location would you like to use? 5
[INFO] Using 'ssd256_2' for storage location.
[INFO] Container ID is 102.
Updating LXC template list...
Downloading LXC template...
Do you want a privileged container?(yes/no)
yes
Creating Privileged LXC container...
Starting LXC container...
Setting up container OS...
Updating container OS...
E: Reading first line of patchfile /var/lib/apt/lists/partial/ftp.debian.org_debian_dists_bullseye-updates_main_binary-amd64_Packages.ed failed
Installing necessary components...
Installing dependencies...
[ERROR:LXC] 100@42 Unknown failure occured.
[ERROR] 100@130 Unknown failure occured.

Xitee1 commented:
@cagnulein 
I am sorry but I'm not really an expert here. I have just updated it to the official installation method of MotionEye because the one from this script was causing errors and updated the Debian version from 10 to 11.
Then it worked for me.

yaba commented:
> @Xitee1 i tried your patch but it didn't work
> 
> root@pvei7:~# bash -c "$(wget -qLO - https://raw.githubusercontent.com/Xitee1/proxmox_motioneye_container/master/create_container.sh)"
> 
> More than one storage locations detected.
> 
> 1. backup
> 2. local
> 3. local-lvm
> 4. ssd256
> 5. ssd256_2
> 6. ssd480
> 
> Which storage location would you like to use? 5 [INFO] Using 'ssd256_2' for storage location. [INFO] Container ID is 102. Updating LXC template list... Downloading LXC template... Do you want a privileged container?(yes/no) yes Creating Privileged LXC container... Starting LXC container... Setting up container OS... Updating container OS... E: Reading first line of patchfile /var/lib/apt/lists/partial/ftp.debian.org_debian_dists_bullseye-updates_main_binary-amd64_Packages.ed failed Installing necessary components... Installing dependencies... [ERROR:LXC] 100@42 Unknown failure occured. [ERROR] 100@130 Unknown failure occured.

Worked for me also, I've forked it and double checked: https://github.com/yaba/proxmox_motioneye_container
Remember to update PVE first.
